server/webhook.go

```diff
@@ -691,17 +723,27 @@ func (s *Server) prepareIssueFromPushEventDDL(ctx context.Context, repo *api.Rep
 			)
 			return nil
 		}
-
-		for _, database := range databases {
-			updateSchemaDetails = append(updateSchemaDetails,
-				&api.UpdateSchemaDetail{
-					DatabaseID: database.ID,
-					Statement:  content,
-				},
-			)
+		for _, task := range taskList {
```

Comment:
What if len(taskList) > 1?

Reply:
Will patch them all. This is not possible for now, because there should be only one ddl/dml task for a non-tenant pipeline.

Reply:
We may want to fail if length > 1 as that’s more like fatal error?

Reply:
SG.